### DISCUSSION TITLE
What happens if Kong pod crashed before writing cache invalidation cluster event to DB ?

### DISCUSSION THREAD

**Discussion by shettyh** [Kong/kong#8132] [Help] [answered]
How the cache is invalidated on other pods in this case ?

// @kikito

**Comment by kikito** [comment, marked as answer]
Hello, 

The short answer is that if a Kong node crashes before writing cache invalidation cluster events, then the cache will not be invalidated in other nodes.

Keep in mind however that individual request's light threads can "crash" (result in various forms of errors) with the worker still able to operate and send cache invalidations. It would have to be a very ..."deliberate" shutdown (worker processes on the OS would have to be restarted, they would change PIDs). A server timing out or returning an unexpected status would not be enough to invalidate cluster events.
